src/Container/AssertionPluginManagerFactory.php

```diff
@@ -38,9 +39,17 @@ class AssertionPluginManagerFactory implements FactoryInterface
      */
     public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $config        = $container->get('config')['zfc_rbac']['assertion_manager'];
-        $pluginManager = new AssertionPluginManager($container, $config);
+        $config = $container->get('config')['zfc_rbac']['assertion_manager'];
```

Comment:
@danizord there it is https://github.com/basz/zfc-rbac/blob/master/config/module.config.php#L88